cmd/bundle.rb

```diff
@@ -107,6 +107,11 @@ def bundle
             zap:    args.zap?,
           )
         end
+      when "add"
+        Bundle::Commands::Add.run(
+          global: args.global?,
+          file:   args.file
+        )
```

Comment:
Do you want these commands to be alphabetized? Looks like they're just in any order.

Reply:
any order is fine 👍🏻